python/cudf/cudf/core/column/column.py

```diff
@@ -150,13 +150,9 @@ def equals(self, other):
         return self.binary_operator("eq", other).min()
 
     def all(self):
-        if self.null_count != 0:
-            return False
```

Comment:
Why can we remove these?

Reply:
So this is because, in pandas case if the entire row/column is NA and `skipna` is True, then the result will be False, as for an empty row/column. If `skipna` is False, then NA are treated as True, because these are not equal to zero. So for this reason if we had the above check inplace we had to fill series with some valid values something like this: `series.fillna(series._column.default_na_value())` and then invoke `column.any/all`